### PR TITLE
Kokoru travis

### PR DESCRIPTION
**Pull request by codepawpaw**
Add travis.yml

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    sources:
+      - google-chrome
+    packages:
+      - google-chrome-stable
+node_js:
+  - "7"
+before_install:
+  - npm setup
+install:
+  - webpack
```
Comment (bxcodec) [inline .travis.yml:14]: Pardon me, is webpack will installed automatically ? 
I don't see any travis command to enable this ?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+addons:
+  apt:
+    sources:
+      - google-chrome
+    packages:
+      - google-chrome-stable
+node_js:
+  - "7"
+before_install:
+  - npm setup
```
Comment (bxcodec) [inline .travis.yml:12]: is this command exist ? 

If I was not too old right now, I think this command still  : `npm install`
Or is npm already add new function ?? 😱

**Review by bxcodec — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (codepawpaw) [reply .travis.yml:12]: npm setup is a script function that already there in package.json.
Please see all change.
It should already there in package.json

File: .travis.yml (hunk 1 above)
Comment (codepawpaw) [reply .travis.yml:14]: Sorry there's typo in the install command. You can refer to the newest one

Comment (codepawpaw) [reply .travis.yml:14]: When you run yarn setup, it will automatically installing the webpack

**Pull request closed without merging**